crashlytics/src/cpp/android/crashlytics_android.cc

```diff
@@ -62,19 +62,25 @@ METHOD_LOOKUP_DEFINITION(firebase_crashlytics,
                          CRASHLYTICS_METHODS, CRASHLYTICS_FIELDS)
 
 // clang-format off
+#define CRASHLYTICS_CORE_METHODS(X)                                          \
+  X(LogFatalException, "logFatalException",                                  \
+    "(Ljava/lang/Throwable;)V",                                              \
+    util::kMethodTypeInstance)
+
 #define CRASHLYTICS_CORE_FIELDS(X)                                           \
   X(DataCollectionArbiter, "dataCollectionArbiter",                          \
   "Lcom/google/firebase/crashlytics/internal/common/DataCollectionArbiter;", \
   util::kFieldTypeInstance)
 
 // clang-format on
 METHOD_LOOKUP_DECLARATION(crashlytics_core,
-                          METHOD_LOOKUP_NONE, CRASHLYTICS_CORE_FIELDS)
+                          CRASHLYTICS_CORE_METHODS,
```

Comment:
what the reason of making this change?

Reply:
The logFatalException method is an internal api of CrashlyticsCore, not a public api of Crashlytics. So we added it here so we can call it via jni.